lib/app-oclif.ts

```diff
@@ -43,9 +42,11 @@ export async function run(command: string[], options: AppOptions) {
 				return require('@oclif/command/flush');
 			}
 		},
-		(error: Error) => {
+		error => {
 			// oclif sometimes exits with ExitError code 0 (not an error)
-			if (error instanceof ExitError && error.oclif.exit === 0) {
+			// (Avoid `error instanceof ExitError` here for the reasons explained
+			// in the CONTRIBUTING.md file regarding the `instanceof` operator.)
+			if (error.oclif?.exit === 0) {
```

Comment:
Here I have not used the `instanceOf` function (from `lib/errors.ts`) because the previous code relied on the plain `instanceof` operator as a [type guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#instanceof-type-guards) that allowed the expression `error.oclif.exit` to be compiled. Without the type guard, typescript would not allow writing `error.oclif`. Shame. The new node does not look too bad, though...